src/api/app/views/layouts/webui/webui.html.erb

```diff
@@ -123,6 +127,10 @@
             { 'openSUSEs OBS Portal' => 'http://en.opensuse.org/Portal:Build_service' },
             { 'Reporting a Bug'   => 'http://openbuildservice.org/support/' } ]
 
+          if Announcement.last.present?
+            @helplinks << { 'Terms of Service' => announcement_path(Announcement.last) }
```

Comment:
Do we want to stick to ``Terms of Service`` or use a more generic name (like we already do internally with announcement)? I would name the link maybe ``Read full announcement``?

Reply:
I would stick with Terms of Service since that's what it is right now.

Reply:
@ChrisBr @openSUSE/open-build-service we decided to call it Announcement, but then we decided to focus on ToS only.

We would have to change many things in order to handle many announcements: show a list of announcements in the "flash", let the user accept them one by one, keep showing those which are not accepted yet..

I think it is not the point right now.  Maybe replacing "Announcement" name by "TermsOfService" has more sense?

Reply:
This is _NOT_ about terms of service. This is about informing people about $SOMETHING.

Just drop this link...

Reply:
The story is even called [Terms of Service](https://trello.com/c/Py3l0qbS/823-p5-terms-of-service-dialog). You are reinterpreting this feature on the fly:wink:

Reply:
That said, I agree with dropping the link for the sake of simplicity

Reply:
The title of the story is 'Terms of Service', the story itself says nothing about it. It's just a name...

Reply:
The card says: "The OBS admin needs a way to show some text for the users, which they need to acknowledge." and almost each acceptance criteria refer to Terms of Service:-)

Like:
```
* Admins can create ToS and update them via API
* The ToS dialog is shown until a user acknowledge the ToS
...
```

Reply:
Sure it refers to the name we have given it. Anyway, this is not something we can solve by PR comments...